src/workloads/scale/BulkLoading.yml

```diff
@@ -0,0 +1,287 @@
+            Operations:
+              - OperationName: insertOne
+                OperationCommand:
+                  Document: *Document
+
+Actors:
+  # This loads about 35 GB of data to warm up the DB
+  - ActorFromTemplate:
+      TemplateName: BulkWriterTemplate
+      TemplateParameters:
+        Name: Warmup
+        Threads: 20
+        OnlyActiveInPhase: [0, 4, 9, 13, 18, 22, 27]
+        Duration: 2 minutes
+        Operations: *WriteOps100x
+
+  - Name: BuildIndexActor
+    Type: RunCommand
+    Threads: 1
+    Phases:
+      OnlyActiveInPhases:
+        Active: [5, 14, 23, 28]
+        NopInPhasesUpTo: *NumPhases
+        PhaseConfig:
+          Repeat: 1
+          Database: *DbName
+          Operations:
+            - OperationMetricsName: BulkBuildColumnStoreIndex
+              OperationName: RunCommand
+              OperationCommand:
+                createIndexes: *CollectionName
+                indexes:
+                  - key: { sensorId: 1, timestamp: -1 }
+                    name: SensorAndTimestamp
+                  - key: { timestamp: -1 }
+                    name: Timestamp
+
+  - Name: QuiesceActor
+    Type: QuiesceActor
+    Threads: 1
+    Database: *DbName
+    Phases:
+      OnlyActiveInPhases:
+        Active: [3, 8, 12, 17, 21, 26, 31]
+        NopInPhasesUpTo: *NumPhases
+        PhaseConfig:
+          Repeat: 1
+
+  - Name: DeleteCollectionActor
+    Type: RunCommand
+    Threads: 1
+    Phases:
+      OnlyActiveInPhases:
+        Active: [2, 7, 11, 16, 20, 25, 30]
+        NopInPhasesUpTo: *NumPhases
+        PhaseConfig:
+          Repeat: 1
+          Database: *DbName
+          Operations:
+            - OperationName: RunCommand
+              OperationCommand:
+                drop: *CollectionName
+
+  - ActorFromTemplate:
+      TemplateName: BulkWriterTemplate
+      TemplateParameters:
+        Name: BulkWriteSingleConnectionNoIndex
+        Threads: 1
+        OnlyActiveInPhase: [1]
+        Duration: 1 minute
+        Operations: *WriteOps100x
+
+  - ActorFromTemplate:
+      TemplateName: BulkWriterTemplate
+      TemplateParameters:
+        Name: BulkWriteSingleConnectionWithIndex
+        Threads: 1
+        OnlyActiveInPhase: [6]
+        Duration: 1 minute
+        Operations: *WriteOps100x
+
+  - ActorFromTemplate:
+      TemplateName: BulkWriterTemplate
+      TemplateParameters:
+        Name: BulkWriteManyConnectionsNoIndex
+        Threads: 20
+        OnlyActiveInPhase: [10]
+        Duration: 1 minute
+        Operations: *WriteOps100x
+
+  - ActorFromTemplate:
+      TemplateName: BulkWriterTemplate
+      TemplateParameters:
+        Name: BulkWriteManyConnectionsWithIndex
+        Threads: 20
+        OnlyActiveInPhase: [15]
+        Duration: 1 minute
+        Operations: *WriteOps100x
+
+  - ActorFromTemplate:
+      TemplateName: NormalWriterTemplate
+      TemplateParameters:
+        Name: NormalWriteManyConnectionsNoIndex
+        Threads: 30
```

Comment:
What's the rationale for different thread levels?

Reply:
Added a comment describing this.